README.md

```diff
@@ -56,7 +56,8 @@ Version | SonarQube | sonar-scala | sonar-scala-extra
 [3.2.0](https://github.com/mwz/sonarqube-scala-docker/releases/tag/3.2.0) | 7.4 ([documentation](https://docs.sonarqube.org/7.4))([changelog](https://jira.sonarsource.com/jira/secure/ReleaseNote.jspa?projectId=10930&version=14549)) | [7.2.0](https://github.com/mwz/sonar-scala/releases/tag/v7.2.0)
 [3.1.0](https://github.com/mwz/sonarqube-scala-docker/releases/tag/3.1.0) | 7.4 ([documentation](https://docs.sonarqube.org/7.4))([changelog](https://jira.sonarsource.com/jira/secure/ReleaseNote.jspa?projectId=10930&version=14549)) | [7.1.0](https://github.com/mwz/sonar-scala/releases/tag/v7.1.0)
 [3.0.0](https://github.com/mwz/sonarqube-scala-docker/releases/tag/3.0.0) | 7.3 ([documentation](https://docs.sonarqube.org/display/SONARQUBE73/Documentation))([changelog](https://jira.sonarsource.com/jira/secure/ReleaseNote.jspa?projectId=10930&version=14464)) | [7.0.0](https://github.com/mwz/sonar-scala/releases/tag/v7.0.0)
-[2.10.0](https://github.com/mwz/sonarqube-scala-docker/releases/tag/2.10.0) | 6.7.6 LTS ([documentation](https://docs.sonarqube.org/display/SONARQUBE67/Documentation))([changelog](https://jira.sonarsource.com/jira/secure/ReleaseNote.jspa?projectId=10930&version=13972)) | [6.7.0](https://github.com/mwz/sonar-scala/releases/tag/v6.7.0)
```

Comment:
You may want to be consistent with the last `|`, sometimes you add it, sometimes you don't.
And even if Markdown is smart enough to do the correct thing for both cases, IMHO, is better to be consistent.

Since this file is being generated, I assume it has to be something in the template.
Apparently it is not being added for any of the `3.x` release, neither for the latest `2.x` release. But, it is added for all of the previous releases.

Reply:
It might be a bit confusing, but some of the lts versions have a version in the additional column for `sonar-scala-extra`, so the trailing pipe isn't actually a closing pipe, but it indicates an empty column. Since all of the lts versions (except for the latest) are generated in a sequence using the same row template, there is an empty column for rows starting from 2.7.0. To be consistent and not have this trailing pipe, I'd need to split out the older releases and have them in a separate array in vars.js and a separate row template just for those few releases, so it doesn't feel to me like it's worth the effort for now. The list is growing though, so at some point, I'll probably move it to a separate file, and perhaps then it will be worth to refactor this.